src/integration/pages/frontPage/__snapshots__/arabic.test.js.snap

```diff
@@ -253,6 +253,12 @@ exports[`Given I am on a Arabic Frontpage AMP/Canonical page When I view the sou
     >
       {"@context":"http://schema.org","@type":"WebPage","url":"http://localhost:7080/arabic","publisher":{"@type":"NewsMediaOrganization","name":"BBC News عربي","publishingPrinciples":"https://www.bbc.com/arabic/institutional-49283069","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://news.files.bbci.co.uk/ws/img/logos/og/arabic.png"}},"image":{"@type":"ImageObject","width":1024,"height":576,"url":"https://news.files.bbci.co.uk/ws/img/logos/og/arabic.png"},"thumbnailUrl":"https://news.files.bbci.co.uk/ws/img/logos/og/arabic.png","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:7080/arabic","name":"الرئيسية"}}
     </script>
+    <script data-react-helmet="true"
```

Comment:
I'm just wondering why this is rendering on the arabic front page, when we shouldn't be showing ads at all on the arabic service?

Reply:
I've just checked the confing - it looks like the arabic service is misconfigured:

```
ads: {
  hasAds: arabic,
}
```

This looks like a typo - it should be `false`. Could you update this? :)

Reply:
Thanks for catching this 👍. Updated the config and the snapshot.